src/SFA.DAS.AdminService.Web/Helpers/UserViewModelHelper.cs

```diff
@@ -0,0 +1,36 @@
+﻿using AutoMapper;
+using SFA.DAS.AdminService.Web.ViewModels.Register;
+using SFA.DAS.AssessorService.Application.Api.Client.Clients;
+using System;
+using System.Threading.Tasks;
+using static StackExchange.Redis.Role;
+
+namespace SFA.DAS.AdminService.Web.Helpers
+{
+    public class UserViewModelHelper : IUserViewModelHelper
```

Comment:
This interface can go since it's not needed any more. Other than that, looks good to me

Reply:
Had a chat with Chris about this. We have very similar ideas for the refactor and the difference is mostly terminology. Resolving